Felienne/hedy
# fixed indentation lvl 15 rock and lvl 3 restaurant NL

**Description**
After finding bug #2420, I have found two similar cases. In two levels, 3 and 15 there were indentation issues. Due to this, the example code of level 15: rock adventure would not work. In level 3 the "```" were visible in the explanation text.

**Fixes #2425**

**How to test**
1. Go to level 3 in Dutch, the restaurant adventure: confirm that the ``` are no longer present. 
2. Go to level 15 in Dutch, the rock adventure: confirm that the example code works.
